frontend/apps/marketing/src/components/error/error.module.scss

```
@@ -0,0 +1,22 @@
+.errorContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 10px;
```

Can you update this to match the [mock](https://www.figma.com/design/l7PsQAs2pitCNP8zUHWEuy/DSCO-CMS?node-id=6079-1980&t=wPFyM2yY7n09PHGc-1):
```suggestion
  padding: 6.25rem 1rem;
```

---

Can you update the screenshots above to show this change?

---

Updated the screenshots!